solution-base/kafka/Makefile

```diff
@@ -0,0 +1,23 @@
+ROOT_DIR:=$(shell dirname $(realpath $(lastword $(MAKEFILE_LIST))))
```

Comment:
Thanks for doing this one, it's simple and I hope it will be very helpful for future maintenance :)